packages/pci-sdk-iframe/src/pages/app/components/Notification/Notification.css

```diff
@@ -0,0 +1,50 @@
+.Notification {
+	display: flex;
+	position: absolute;
+	z-index: 3;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	align-items: center;
+	justify-content: flex-start;
+	padding: 4.7vw;
+	animation: slide-up 0.6s ease-in;
+	border-radius: 0 0 12px 12px;
+	color: white;
+	font-family: sans-serif;
+	font-size: 4.118vw; /* 14px */
+	font-weight: 600;
+	line-height: 4.7vw;
+	text-align: left;
+}
+
+.Notification--positive {
+	background-color: #00a96d;
+}
+
+.Notification--negative {
+	background-color: #b94128;
+}
+
+.Notification__icon {
+	flex-shrink: 0;
+	width: 28px;
+	height: 30px;
+	margin-right: 4px;
+}
+
+.Notification__icon--positive {
+	background-image: url('./images/positive-icon.svg');
+}
+.Notification__icon--negative {
+	background-image: url('./images/negative-icon.svg');
+}
+
+@keyframes slide-up {
+	from {
+		bottom: -100%;
+	}
+	to {
+		bottom: 0;
+	}
+}
```

Comment:
For [performance](https://stackoverflow.com/questions/13410723/transformtranslatex-vs-transition-on-left-property-which-has-better-performanc), we might want to do this with `transform: translateY(-100%)` instead.

IIRC `transform` does better on mobile devices and it seems we're moving the element quite a lot here.